common/Generieke-Datatypen-Gemeenten.yaml

```diff
@@ -0,0 +1,79 @@
+openapi: 3.0.0
```

Comment:
We hebben nu een 'Generieke Datatypen Gemeenten' model op zowel SIM als UGM en BSM niveau. Volgens mij is dat helemaal niet nodig zoals dat ook voor 'MIM11 Primitieve datatypen' niet nodig is.

Enige bezwaar daarbij is dat in het SIm model 'Generieke Datatypen Gemeenten' een attribute 'Lokaal Id' tegen komen terwijl er in het UGM en BSM geen spaties in de attributenamen worden opgenomen.

Reply:
Ik heb in het Json-schema een toelichting opgenomend dat het schema gebaseerd is op het SIM Generieke Datatypen Gemeenten, maar dat we wel de naamgevingesconventie van het BSM gebruiken.

Reply:
Top